h

```diff
@@ -0,0 +1,1022 @@
+[33mcommit f21266c789f59ed0f390f73bead9b1c73cf9b250[m[33m ([m[1;36mHEAD -> [m[1;32mbranch_bug_rectify12[m[33m)[m
```

Comment:
What's this file?

Reply:
Oh. Don't know how it came. Corrected it. Thanks